src/Ionide.ProjInfo/Library.fs

```diff
@@ -505,7 +585,7 @@ module ProjectLoader =
             Init.setupForLegacyFramework msbuildBinaryDir
         | _ -> ()
 
-    let loadProject (path: string) (binaryLogs: BinaryLogGeneration) globalProperties =
+    let loadProject (path: string) (binaryLogs: BinaryLogGeneration) (projectCollection: ProjectCollection) =
```

Comment:
a lot of these calls dropped `globalProperties` in favor of `projectCollection` because the projectCollection _holds_ the global properties.